lms/djangoapps/ccx/views.py

```diff
@@ -540,3 +543,24 @@ def ccx_grades_csv(request, course, ccx=None):
         response['Content-Disposition'] = 'attachment'
 
         return response
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@coach_dashboard
+def rename_ccx(request, course, ccx=None):
+    """
+    Rename ccx display name.
+    """
+    if not ccx:
+        raise Http404
+
+    display_name = request.POST.get('name')
+
+    if display_name:
+        ccx.display_name = display_name
+        ccx.save()
+
+        return JsonResponse({"status": _("ok")})
+
+    return JsonResponse({"status": _("Unable to save display name, name parameter must be a valid string")})
```

Comment:
@giocalitri  will you provide API for this task or already their is API for this?
https://github.com/mitocw/edx-platform/pull/163#discussion_r49738741

cc @pdpinch